dcmio/dcmconverter/dicom_utils.py

```diff
@@ -99,10 +100,22 @@ def add_meta_to_nii(nii_files, dicom_dir, prefix, dcm_tags, output_directory,
                 header.set_slice_duration(slice_duration)
 
             # > add free dicom fields
-            content = dict(
-                (str(name), str(dcmimage[tag].value))
-                for name, tag in dcm_tags)
-
+            # extract value from the dicom file
+            content = {}
+            for name, tag in dcm_tags:
+
+                try:
+                    # enhances storage, the value is burried under one or
+                    # several layer(s) of sequence
+                    dcmimage_temp = dcmimage
+                    if len(tag) > 1:
+                        for sub_tag in tag[:-1]:
+                            seq_field = dcmimage_temp[sub_tag]
+                            dcmimage_temp = seq_field.value[0]
```

Comment:
SQ is the only element from which we can go deeper. If several tags are set for one value to extract, we assume all intermediate tags point to sequence (SQ) fields.

We take the first inner sequence. So far the information we need if enclosed in ALL inner sequences (<Sequence, length 5550, at 7FCE7A5E7C00>: explore all sequences can waste time).